localbuild/meta.yaml

```diff
@@ -47,7 +47,6 @@ requirements:
     - netcdf4
     - hdf4
     - pillow
-    - pytz
```

Comment:
this was in the past wrong to add it here

Reply:
ok